### PR TITLE
Update: update file picker button to inverse button

### PR DESCRIPTION
**Pull request by pphminions**
Change the file picker button to inverse button (it was a primary button)

screen shot 2016-07-22 at 3 49 41 pm

https://trello.com/c/VGbO8Ut3/5525-briefing-attatchments-button

### CODE REVIEW

**Comment by pphminions** [comment]
@omgaz ran `npm i ` before pushing the code to the branch :)

**Comment by sharmajyo** [comment]
LGTM

**Comment by NamrataTrivedi** [comment]
LGTM :candy:

**Comment by omgaz** [comment]
LGTM

**Comment by omgaz** [comment]
[LGTM](http://lgtm.in/i/GHdgaCZdq)

**Comment by pphminions** [comment]
@omgaz please merge if the code is okay :)

**Comment by omgaz** [comment]
Merging 🍜

File: src/components/adslotUi/FilePickerComponent.js [hunk 1]
```diff
@@ -42,7 +42,7 @@ class FilePickerComponent extends React.Component {
         <div className="input-group-btn">
           {fileName ? <Button className="remove-file btn-inverse" onClick={this.removeFile}>×</Button> : null}
```
Comment (OndrejCholeva) [inline src/components/adslotUi/FilePickerComponent.js]: are you keen fixing this one as well? https://trello.com/c/bmd0ibrR/5490-filepicker-remove-button-style-is-broken

**Pull request merged**